fhi_users/auth/rest_auth_views.py

```
@@ -352,6 +357,12 @@ def perform_create(
         new_domain: bool = bool(data["make_new_domain"])  # type: ignore
         user_domain_opt: Optional[UserDomain] = None
 
+        if not auth_utils.validate_password(user_signup_info["password"]):
```

Ruff (0.8.2)

Undefined name `auth_utils`

(F821)